Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => {
+    
+    return (
+        <footer>
+            <div class='image'>
+                <img class='chat' src={require('./speech-bubble.png')}/>
```

Comment:
Note that instead of using images, you can also get most of these icons through the font awesome library

Reply:
do I have to import that library?  if so, how do I use the fonts?